patternfly/patternfly-elements
# fix: Fix disabled CTA rendering

## pfe-cta

Fix disabled CTA styles for IE11:

<img width="1204" alt="Screen Shot 2020-07-21 at 9 51 35 AM" src="https://user-images.githubusercontent.com/1840295/88063524-1d653300-cb38-11ea-97dd-c9af15562a5e.png">

### Related issue

- (#991) IE11: pfe-cta



### What has changed and why

Add fallback styles for IE11 disabled button, comment out var stack that is causing IE11 issue: `FontSize--priority`.


### Testing instructions

1. Open the demo page in IE11.
2. Ensure that the disabled links have styles.


#### Browser requirements

Your component should work in all of the following environments:


### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
